Description: allow parser to build even when not on Linux.
Author: Kees Cook <kees@debian.org>

Index: apparmor-2.10.95/common/Make.rules
===================================================================
--- apparmor-2.10.95.orig/common/Make.rules
+++ apparmor-2.10.95/common/Make.rules
@@ -85,7 +85,7 @@ pod_clean:
 CAPABILITIES=$(shell echo "\#include <linux/capability.h>" | cpp -dM | LC_ALL=C sed -n -e '/CAP_EMPTY_SET/d' -e 's/^\#define[ \t]\+CAP_\([A-Z0-9_]\+\)[ \t]\+\([0-9xa-f]\+\)\(.*\)$$/CAP_\1/p' | LC_ALL=C sort)
 
 .PHONY: list_capabilities
-list_capabilities: /usr/include/linux/capability.h
+list_capabilities:
 	@echo "$(CAPABILITIES)"
 
 # =====================
